### PR TITLE
Fix pinner

### PR DESCRIPTION
**Pull request by franckc**
### Description:

 - fixed a bug where the event's name was not properly accessed from the data in the message
 - per Protocol Labs' request, added a ?name=OriginProtocol to our requests
 - various clean ups

### Checklist:

- [X] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

File: devops/cloud-functions/gcf-pin-ipfs-hash/index.js [hunk 1]
```diff
@@ -37,7 +39,7 @@ const pinService = async event => {
         // interval every try. Set fields accordingly.
         let numberOfRetries = 4
         let initialRetryInterval = 500
-        let retryIntervalGrowthRate = 2
+        const retryIntervalGrowthRate = 2
```
Comment (mikeshultz) [inline devops/cloud-functions/gcf-pin-ipfs-hash/index.js]: Nit: Perhaps this should be a file-level constant.

File: devops/cloud-functions/gcf-pin-ipfs-hash/ipfs-cluster-api-service.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
 const fetch = require('cross-fetch')
 
+// Per Protocol Labs request, we add this param to any request we send to their cluster.
+const ORIGIN_PARAM = '?name=OriginProtocol'
+
 class IpfsClusterApiService {
   constructor(ifpsClusterUrl, username, password) {
     this.ifpsClusterUrl = ifpsClusterUrl
```
Comment (tomlinton) [inline devops/cloud-functions/gcf-pin-ipfs-hash/ipfs-cluster-api-service.js]: We might as well fix this typo in this PR too?

Comment (franckc) [reply devops/cloud-functions/gcf-pin-ipfs-hash/ipfs-cluster-api-service.js]: Yep. might as well ! done

File: devops/cloud-functions/gcf-pin-ipfs-hash/index.js (hunk 1 above)
Comment (franckc) [reply devops/cloud-functions/gcf-pin-ipfs-hash/index.js]: Good point. Done.

**Pull request merged**